src/dune_util/dune_flock.c

```diff
@@ -3,17 +3,53 @@
 #include <caml/threads.h>
 #include <caml/unixsupport.h>
 
-#ifndef _WIN32
+#ifdef _WIN32
+
+#define FD_val(value) Handle_val(value)
+
+CAMLprim value dune_flock_lock(value v_fd, value v_block, value v_exclusive) {
+  CAMLparam2(v_fd, v_block);
```

Comment:
Why did you skip `v_exclusive`?

Reply:
Good question: actually I copy-pasted from the Unix implementation below. I will add it (in both places). Or we could remove `v_block` from both places (both `v_block` and `v_exclusive` are booleans).

Reply:
Fixed.